webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart

```diff
@@ -30,15 +30,17 @@ class StandardMessagesManager {
       var currentIndex = category.categoryIndex ?? 0;
       return maxIndex > currentIndex ? maxIndex : currentIndex;
     });
-    return lastIndex + 1;
+    var length = localCategories.values.length;
+    return lastIndex >= length ? length : lastIndex + 1;
```

Comment:
is the below equivalent? if so, let's import the `math` package and use `min()` - I think it's making the logic easier to read:
```suggestion
    return min(localCategories.values.length, lastIndex + 1);
```

(if this works, can you apply it to the below lines as well?)